lib/airtime.js

```diff
@@ -49,19 +49,49 @@ class Airtime {
         });        
     }
 
+     findTransactionStatus(transactionId){
+        return new Promise((resolve, reject) => {
+            if(transactionId === undefined){
+                throw new Error(`transactionId should be provided`);
+            }
+            const username = this.config.username;
+            const apikey = this.config.apiKey;
+            
+            axios({
+                method: 'get',
+                maxBodyLength: Infinity,
+                url: `${Common.AIRTIME_URL}/query/transaction/find?username=${username}&transactionId=${transactionId}`,
```

Comment:
`Common.AIRTIME_URL` is `baseUrl + "/airtime/send"`, this will result in a `404`